CONTRIBUTING.md

```diff
@@ -8,6 +8,26 @@ We strongly encourage everyone interested to participate:
 
 Please read our [contributor guide](https://nix.dev/contributing/documentation) for more details.
 
+## Local preview
+
+Enter the development environment with `nix-shell`, or [set up direnv](https://nix.dev/guides/recipes/direnv.html)` and run `direnv allow`, and then:
+
+```shell-session
+[nix-shell:nix.dev]$ devmode
+```
+
+and open a browser at <http://localhost:5500>.
+
+As you make changes, your browser should auto-reload within a few seconds.
+
+To manually test [redirects](./_redirects):
+
+```shell-session
+[nix-shell:nix.dev]$ nix-build -A build
+[nix-shell:nix.dev]$ cd result
+[nix-shell:nix.dev]$ netlify dev
```

Comment:
Actually, that doesn't work, `cd`ing into `result` unloads direnv!

Reply:
```suggestion
[nix-shell:nix.dev]$ netlify dev -d result
```

Cool, thanks a lot!